samcli/lib/package/packageable_resources.py

```diff
@@ -517,6 +518,27 @@ class CloudFormationResourceVersionSchemaHandlerPackage(ResourceZip):
     PROPERTY_NAME = RESOURCES_WITH_LOCAL_PATHS[AWS_CLOUDFORMATION_RESOURCEVERSION][0]
 
 
+class ECRResource(Resource):
```

Comment:
Can you add a docstring explaining what this class is for?
Seems like this class is not for exporting at all. Only for deleting